frontend/src/features/projects/dispose/forms/disposalYupSchema.ts

```diff
@@ -46,7 +46,6 @@ export const SelectProjectPropertiesStepYupSchema = Yup.object().shape({
 });
 
 export const ProjectDraftStepYupSchema = Yup.object().shape({
-  projectNumber: Yup.string().required('Required'),
```

Comment:
As it is generated this would previously stop the user from hitting save/next after entering a name. Worked before as the project was created right away.